test/linalg/symmetric.jl

```diff
@@ -250,3 +250,36 @@ let a = randn(2,2)
     cc = copy(c)
     @test conj!(c) == conj(Array(c))
 end
+
+# 19225
+let X = sparse([1 -1; -1 1])
```

Comment:
I think these should to into the sparse tests for modularity. However, you might want to test the functionality with dense arrays as well.